opta/commands/apply.py

```diff
@@ -129,6 +132,11 @@ def _apply(
     stdout_logs: bool = True,
     detailed_plan: bool = False,
 ) -> None:
+    if is_module_api_enabled():
+        opts = ApplyOptions(auto_approve=auto_approve, config_path=config)
+        apply2(opts)
```

Comment:
can we display a warning in the like - _not for production use_ or _beta feature_ to prevent unintentional use until it's ready?
(probably inside the new process.apply function itself)

Reply:
Added a warning log in the `process.apply` function.